### DISCUSSION TITLE
Flight log for Crazyflie

### DISCUSSION THREAD

**Discussion by an949na** [bitcraze/discussions#1683] [Q&A] [answered]
Hi, Bitcraze community, I ran my Crazyflie 2.1 flies well with 8 shape trajectory, using Lighthouse Positioning System. So now I want to see my drone's actual flight log (for example x, y, z, roll, yaw, pitch, etc.). With Cfclient, it was easy to check this, but I have big struggle in python system because of my bad python level. Help me please!

here is code i used.

https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/autonomy/autonomous_sequence_high_level.py

**Comment by gemenerik** [comment]
Hi! Check out [this step-by-step user guide](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/). That will help you set-up logging.

**Reply by an949na** [reply]
Thank you for your fast reply! In your link, it seems provide log data only about  yaw, pitch, roll. I want to get x, y, z data. Those estimated_data set also provides localization data to?

**Reply by gemenerik** [reply, marked as answer]
You can log any of the logging parameters by adding them to the logging config. In the example I shared, the logging config is defined [here](https://www.bitcraze.io/documentation/repository/crazyflie-lib-python/master/user-guides/sbs_connect_log_param/#add-logging-config). If you don't know the name of the log parameter you want to add, I would follow the guide's recommendation and find the name in the cfclient log TOC tab or in the logging configurations window.
